pkg/utils/nodes.go

```diff
@@ -24,3 +26,15 @@ func IsNodeNameValid(r client.Reader, nodeName string) (bool, error) {
 	}
 	return true, nil
 }
+
+// getNodeFromFAR returns the unhealthy node reported in the given far CR (by it's name)
+func getNodeFromFAR(r client.Client, far *farV1alpha1.FenceAgentsRemediation) (*corev1.Node, error) {
```

Comment:
nit: I think that the function might be easier to understand if named like
`getNodeWithName(r client.Client, name string) (*corev1.Node, error)`

It is up to the caller give the right name (`far.Name` in this case)

WDYT?